test.sh

```diff
@@ -1,20 +1,19 @@
 set -e
 
+# if greenplum supported (gpfdist exist), then test GP
+if hash gpfdist 2>/dev/null; then
+  echo "Testing Greenplum..."
+  bash test/test_gp.sh $1
 # if command psql exist, then test psql
-if hash psql 2>/dev/null; then
+elif hash psql 2>/dev/null; then
```

Comment:
@rionda It was intended to test Greenplum twice, one with "test_gp.sh" which tests with parallel grounding, the other uses non-parallel grounding. Maybe we could find a better structure for this?